hack/tools/release/notes.go

```diff
@@ -223,53 +225,49 @@ func run() int {
 		}
 	}
 
-	for _, c := range commits {
-		body := trimTitle(c.body)
-		var key, prNumber, fork string
-		prefix, err := getAreaLabel(c.merge)
-		if err != nil {
-			fmt.Println(err)
-			os.Exit(1)
-		}
-		switch {
-		case strings.HasPrefix(body, ":sparkles:"), strings.HasPrefix(body, "✨"):
-			key = features
-			body = strings.TrimPrefix(body, ":sparkles:")
-			body = strings.TrimPrefix(body, "✨")
-		case strings.HasPrefix(body, ":bug:"), strings.HasPrefix(body, "🐛"):
-			key = bugs
-			body = strings.TrimPrefix(body, ":bug:")
-			body = strings.TrimPrefix(body, "🐛")
-		case strings.HasPrefix(body, ":book:"), strings.HasPrefix(body, "📖"):
-			key = documentation
-			body = strings.TrimPrefix(body, ":book:")
-			body = strings.TrimPrefix(body, "📖")
-			if strings.Contains(body, "CAEP") || strings.Contains(body, "proposal") {
-				key = proposals
+	results := make(chan processCommitResult)
+	jobs := make(chan *commit)
```

Comment:
nit
```suggestion
	commitCh := make(chan *commit)
```
(using another noun is a little bit confusing)